infra/ansible/README.md

```diff
@@ -0,0 +1,31 @@
+# TestGround setup via Ansible
+
+## Login to onepassword
+
+Run 'eval $(op signin protocollabs)' to sign into your 1Password account, prior to executing playbooks that need access to secrets, such as `setup.yaml`.
+
+## Useful commands
+
+Perform a quick connectivity test
+
+```bash
+$ ansible all -m ping
```

Comment:
Also needs an `-i inventories/$CLUSTER_NAME` I think...

Reply:
I am still used to writing Ansible/Terraform playbooks for specific machines/clusters, where we set up defaults for inventories, but looks like here it won't be possible, since we'd be using these for multiple setups.

This makes me think that the `terraform backend` is also something that would be PL-specific, as we want a backend only for shared infrastructure, such as the production cluster.